app/views/Application/app.scala.html

```diff
@@ -0,0 +1,9 @@
+@(title: String, jsLocation: String)
```

Comment:
Any reason for which you created a new folder under `views`. Do we really need this extra level of nesting?